### ISSUE TITLE
Teachers see Dashboard of individual SU

### ISSUE THREAD

**Issue by katyedwards** [healthlocker/oxleas-adhd#188]
As a Teacher I can

- [ ] Click on an individual SU in my caseload to see more information

- [ ] Send messages to a SU?

- [ ] View their tracking, diary, goals, strategies - if shared see #182 

- [ ] View their details & contacts page  (Not Meds?)

**Comment by katbow** [comment]
@katyedwards I am going to change this to an epic & break out each section. Some of them (i.e. messaging and the changes to tracking would be a T1d+. I'll link each issue in here after they are made.

**Comment by iteles** [comment]
Added those to the wrong issues, apologies!

**Comment by iteles** [comment]
@katyedwards This epic is in Sprint 3, but the final checkbox issues (#189 and #205) are both in the backlog - is this correct?

**Comment by katyedwards** [comment]
Yep for now that is right - they just want to get something in there for teachers and the structured form offers most flexibility.

**Comment by katyedwards** [comment]
Just #205 remaining now.
